FirefoxPrivateNetworkVPN/Models/VPNLocations.swift

```diff
@@ -52,14 +52,31 @@ struct VPNCity: Codable {
         self.longitude = longitude
         self.servers = servers
         self.flagCode = flagCode
+        self.setServerSelection()
```

Comment:
What do you think about removing this here and only setting it when a city is selected?